Website/src/rendering/src/assets/css/components/Checkout/CartDetails.css

```diff
@@ -147,8 +151,12 @@
     }
 
     .user-comment {
```

Comment:
When focused, the comment and promo inputs have rounded border corners. Could they please be adjusted to stay square as on the checkout page?